lib/active_model/serializer.rb

```diff
@@ -98,6 +98,16 @@ def self.get_serializer_for(klass)
       end
     end
 
+    def self.include_directive_from_options(options)
```

Comment:
missing `@api private`.  These methods are being held here for now, but may move around, be renamed, or disappear